World/Magic/IPowerSource.cs

```diff
@@ -0,0 +1,7 @@
+namespace World.Magic
+{
+    public interface IPowerSource
+    {
+        long consumeTotal(long cost);
```

Comment:
Methodenamen met hoofdletters
```suggestion
        long ConsumeTotal(long cost);
```